### DISCUSSION TITLE
Temporarily disable Renovate?

### DISCUSSION THREAD

**Discussion by janpio** [renovatebot/renovate#10374] [Request Help]
This might be slightly offtopic, but also very on topic as it is about Renovate: 

Does anyone have a good way to disable Renovate for some time? 

Our repo has quite a few tests, and when we are working on big branches and doing a lot of commits GitHub Actions is prone to queue many of our jobs. If then Renovate comes in with updates to 5 dependencies via 5 branches/PRs, that obviously does not help the queue size. Add in our own automation adding new development builds of our internal libraries, this causes all open PRs to be rebased... and everything just implodes.

We would love to have a way to "pause" renovate PR and branch creation for a bit, possibly also the rebasing onto the branch and in general anything not strictly necessary that creates more CI runs and jobs.

Any ideas how we could tackle that?

(Obviously we could remove the Renovate configuration for a bit - but that would be quite a big hammer to use. I am hoping there is something more fine grained.)

**Comment by rarkins** [comment]
`"enabled": false` in the root of the renovate config will cause it to stop but it might also cause it to delete all existing branches. 

Setting a schedule with updateNotScheded=false should mostly disable it. 

You can also suspend the GitHub installation of using the app, but that applies to all repos.

**Reply by janpio** [reply]
`updateNotScheded=false` looks most like what I might be looking for then. 

Can we maybe configure that via an environment variable in the repository or something, or just via a commit to the renovate config?

**Reply by rarkins** [reply]
You can't have environment variables in a repository, it would need to be in the bot server - in which case it applies to all repos. Otherwise in your repo config if you want to make it per-repo

**Reply by janpio** [reply]
Hm true, I thought the "Secrets" might be available - but they only apply to GH Actions, not apps. Sad.

**Comment by Jolg42** [comment]
My thinking is that schedule can be added and set to something like first day of the month.

Or that you can add invalid json like 
````
Invalid, to pause renovate, remove later.
{ The valid config }
````

Not as good as a pause fonctionnality but should work?

**Reply by rarkins** [reply]
Invalid json will result in an issue raised in repos which could be quite noisy
